### PR TITLE
Popover: migrate docs examples to Sandpack

### PR DESCRIPTION
**Pull request by dangerismycat**
Also added one new variant about Popover visibility on page load. Didn't modify content of existing examples, just migrated them as-is

### CODE REVIEW

File: docs/examples/popover/dontCaret.js [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+            anchor={anchorRef.current}
+            id="main-example"
+            idealDirection="down"
+            onDismiss={() => setOpen(false)}
+            positionRelativeToAnchor={false}
+            size="xl"
+            showDismissButton
+            showCaret
+          >
+            <Box width={360}>
```
Comment (AlbertCarreras) [inline docs/examples/popover/dontCaret.js:138]: I'd to 250 here so it displays nicely

File: docs/pages/web/popover.js [hunk 2]
```diff
@@ -162,360 +55,64 @@ function PopoverExample() {
...
-            </Box>
-          </Popover>
-        </Layer>
-      )}
-    </React.Fragment>
-  );
-}
-  `}
+            sandpackExample={
+              <SandpackExample code={main} hideEditor name="Do - Lightweight" previewHeight={400} />
```
Comment (AlbertCarreras) [inline docs/pages/web/popover.js:66]: I'd do 250 in the Box width so it displays nicely.

File: docs/examples/popover/a11y.js [hunk 3]
```diff
@@ -99,6 +99,7 @@ export default function DismissButton(): Node {
           <Button color="red" onClick={() => {}} size="lg" text="Save" />
         </Flex>
       </Box>
+
```
Comment (AlbertCarreras) [inline docs/examples/popover/a11y.js:102]: I'd do 250 here so it displays nicely too.

**Review by AlbertCarreras — APPROVED** [review]

File: docs/examples/popover/dontCaret.js (hunk 1 above)
Comment (dangerismycat) [reply docs/examples/popover/dontCaret.js:138]: I split the difference and went with 300 (250 seemed a tad narrow)

**Pull request merged**